site/slices/contact-us-slice/form/index.js

```diff
@@ -15,6 +15,23 @@ class Form extends Component {
     };
   }
 
+  componentDidMount = () => {
+    const { stateNavigator } = this.context;
+    stateNavigator.onNavigate(this.focusText);
+  }
+
+  componentWillUnmount = () => {
+    const { stateNavigator } = this.context;
+    stateNavigator.offNavigate(this.focusText);
+  }
```

Comment:
The `stateNavigator` is put in the context by the Layout component. Should I pass it as prop to the components?